Demo/Demo/ViewModels/VaultState.swift

```diff
@@ -12,6 +12,7 @@ struct CardVaultState: Equatable {
     var setupToken: SetUpTokenResponse?
     var updateSetupToken: UpdateSetupTokenResult?
     var paymentToken: PaymentTokenResponse?
+    var paypalVaultToken: String?
 
```

Comment:
I am wondering if I should add cancel state in LoadingState. 
I do provide delegate function for cancel so merchant can handle it the way they want
but I didn't include it as a separate state in the demo app because it didn't make sense for me to
display anything for canceling.

Reply:
I think it depends on the Merchant's needs. For now, we can leave it as is; It LGTM. In fact, if you'd like, we can even consider the approach used in the `func cardDidCancel(_ cardClient: CardPayments.CardClient)` within the `CardPaymentViewModel`, where we use the .idle state.